openmc/material.py

```diff
@@ -255,6 +263,18 @@ def fissionable_mass(self):
                            / openmc.data.AVOGADRO
         return density*self.volume
 
+    @property
+    def decay_photon_source(self) -> Univariate:
```

Comment:
```suggestion
    def decay_photon_energy_distribution(self) -> Univariate:
```
Same as above, I think the use of the word source will drive users to think this returns something that can be used in place of openmc.Source